app/controllers/api/v1/sessions_controller.rb

```diff
@@ -6,9 +6,9 @@ def create
       render json: UserSerializer.new(@user)
     else
       render json: { 
-        status: 401,
+        # do we need logged_in: false here?
```

Comment:
I don't think so, as far As I could work out, that would be the default on the Front End...

Reply:
Ok. This is mostly my unfamiliarity with `Ruby`, and where all we need to account for this.